.github/workflows/test-base.yml

```diff
@@ -1,6 +1,6 @@
-name: Test (base)
+name: Test Base
```

Comment:
Can a comment be added to each workflow file to say what it depends on (or what it calls)?